src/lib/form/fieldManager.ts

```diff
@@ -76,10 +89,19 @@ export class FieldManager extends AbstractFormNode {
     });
   }
 
-  public serialise(): FieldJSON {
-    const value = this._value;
-    const errorMessages = this.errorMessages();
+  private replaceKeywords(errorMessage: string): string {
+    const keywords = Object.keys(this._keywordMap);
 
-    return { value, errorMessages };
+    return keywords.reduce((errorMessage, keyword) => {
+      const keywordRegex = new RegExp("%" + keyword + "%", "g");
+
+      if (keywordRegex.test(errorMessage))
+        return errorMessage.replace(
+          keywordRegex,
+          this._keywordMap[keyword](this.value)
+        );
+
+      return errorMessage;
+    }, errorMessage);
```

Comment:
As discussed yesterday, `replaceAll` not available in our version of node, so sticking with using `.replace()` with the `"g"` (global) flag set on the RegExp.

Reply:
`if (keywordRegex.test(errorMessage))` is necessary to avoid `keyword` being undefined in line 101, i.e.

```typescript
this._keywordMap[undefined](this.value)
```

without the boolean test at line 98 results in an error in cases where no keyword is set in the error message.